dsl_parser/functions.py

```diff
@@ -897,3 +908,58 @@ def replace_with_raw_function(*args):
 
     # Change previously replaced get_property instances with raw values
     scan.scan_service_template(plan, replace_with_raw_function, replace=True)
+
+
+def get_nested_capability_attribute(root, path):
+    def _get_error_string(attr_list):
```

Comment:
It's the same as _get_attr_drill_down_string

Reply:
Oops.